index.html

```diff
@@ -12,11 +12,14 @@
   <body>
       <div class="container">
         <img class="book-image" src="./src/img/book-cover.png" alt="Book image">
-        <div>
+        <div class="book-info">
           <h1 class="book-title">Título del libro <span class="book-genre">(género)</span></h1>
-          <p class="book-attribute"><strong class="reset-strong">Autor del libro: </strong>Nombre del autor del libro</p>
-          <p class="book-attribute"><strong class="reset-strong">Editorial: </strong>Nombre de la editorial</p>
-          <p class="book-attribute"><strong class="reset-strong">Año de publicación: </strong>Año de publicación</p>
+          <hr class="book-divider">
```

Comment:
the hr element is used to define a thematic break, in this case the break is only visual, it would be better to use a border